etcd/fs/fs_test.go

```diff
@@ -227,3 +227,59 @@ func TestFs3(t *testing.T) {
 	}
 	t.Logf("tree2: \n%s", tree2)
 }
+
+func TestFs4(t *testing.T) {
+	etcdClient := &etcd.ClientEtcd{
+		Seeds: []string{"localhost:2379"}, // endpoints
+	}
+
+	if err := etcdClient.Connect(); err != nil {
+		t.Logf("client connection error: %+v", err)
+		return
+	}
+	defer etcdClient.Destroy()
+
+	etcdFs := &etcdfs.Fs{
+		Client:     etcdClient.GetClient(),
+		Metadata:   superblock,
+		DataPrefix: etcdfs.DefaultDataPrefix,
+	}
+
+	etcdFs.Mkdir("/tmp", umask)
```

Comment:
@purpleidea I had to add this, since I was reusing /tmp from the previous tests.